modules/taxonomy/plant_type/farm_plant_type.module

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * @file
+ * Contains farm_plant_type.module.
+ */
+
+use Drupal\Core\Entity\Display\EntityFormDisplayInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+
+/**
+ * Implements hook_entity_form_display_alter().
+ */
+function farm_plant_type_entity_form_display_alter(EntityFormDisplayInterface $form_display, array $context) {
+  if ($context['entity_type'] == 'taxonomy_term' && $context['bundle'] == 'plant_type' && $form_display->getMode() == 'default' && $form_display->isNew()) {
+    $form_display->setComponent('crop_family', [
+      'type' => 'entity_reference_autocomplete',
+      'settings' => [
+        'match_operator' => 'CONTAINS',
+        'match_limit' => 10,
+        'size' => 60,
+        'placeholder' => '',
+      ],
+      'region' => 'content',
+      'weight' => 10,
+    ]);
+    $form_display->setComponent('maturity_days', [
+      'type' => 'number',
+      'settings' => [
+        'placeholder' => '',
+      ],
+      'region' => 'content',
+      'weight' => 20,
+    ]);
+    $form_display->setComponent('transplant_days', [
+      'type' => 'number',
+      'settings' => [
+        'placeholder' => '',
+      ],
+      'region' => 'content',
+      'weight' => 15,
+    ]);
+    $form_display->setComponent('companions', [
+      'type' => 'entity_reference_autocomplete',
+      'settings' => [
+        'match_operator' => 'CONTAINS',
+        'match_limit' => 10,
+        'size' => 60,
+        'placeholder' => '',
+      ],
+      'region' => 'content',
+      'weight' => 40,
+    ]);
+    $form_display->setComponent('image', [
+      'type' => 'image_image',
+      'settings' => [
+        'progress_indicator' => 'throbber',
+        'preview_image_style' => 'thumbnail',
+      ],
+      'region' => 'content',
+      'weight' => 50,
+    ]);
```

Comment:
With this PR adding `image` as a base field we shouldn't need to set the `image` component for `plant_type` terms, yes?

Reply:
I think this will fix a minor issue where the ontology field is rendered between images+files, only seeing this on `plant_type` terms

![Screenshot from 2024-03-20 18-02-31](https://github.com/farmOS/farmOS/assets/3116995/b55d2f64-cf6c-41e5-a541-f12326213cad)

Reply:
Oh good catch @paul121! I meant to remove that. :-)

Reply:
Force pushed.